src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/NonGCStaticsNode.cs

```diff
@@ -53,6 +53,14 @@ int ISymbolNode.Offset
             }
         }
 
+        public MetadataType Type
+        {
+            get
+            {
+                return _type;
```

Comment:
Yes I do, but the other part of me wants to maintain the same style as the rest of the file. At some point we should have dotnet-bot make a sweep to change these everywhere.